Adds the compatibility tag and runs the tests on Python 3.9 (#71)
PennyLaneAI/pennylane-sf: .github/CHANGELOG.md

# Release 0.16.0

### Breaking changes

> * Add support and tests for Python 3.9.

Reviewer: Also, would suggest moving this to `Improvements`.
```suggestion
* Adds the compatibility tag and runs the tests on Python 3.9.
```

> * Adds the compatibility tag and runs the tests on Python 3.9.

Reviewer: ```suggestion
* Adds the compatibility tag for Python 3.9.
```
